### PR TITLE
Create an experiment to turn amp-sticky-ad to amp-ad sticky implementation

### PR DESCRIPTION
**Pull request by powerivq**
Create an experiment to turn amp-sticky-ad element into an amp-ad element. In theory they should act the same. Set the experiment group info for Adsense/Doubleclick.

### CODE REVIEW

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js [hunk 1]
```diff
@@ -112,6 +114,41 @@ class AmpStickyAd extends AMP.BaseElement {
...
+    }
+
+    const children = realChildElements(this.element);
+    userAssert(
+      children.length == 1 && children[0].tagName == 'AMP-AD',
+      'amp-sticky-ad must have a single amp-ad child'
+    );
+
+    const ad = children[0];
+    const enableConversion = Math.random() < 0.5;
```
Comment (calebcordry) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js:130]: Do you not want to use the randomlySetBranch helper?

Comment (powerivq) [reply extensions/amp-sticky-ad/1.0/amp-sticky-ad.js:130]: Each page could only have one amp-sticky-ad, so I do not need to persist the branch state on the `window` object. Without that, `randomlySetBranch` is essentially doing the same thing as `Math.random`, and I feel it is clearer than declaring something like

```
{
          experimentId: ZINDEX_EXP,
          isTrafficEligible: () => true,
          branches: Object.values(ZINDEX_EXP_BRANCHES),
}
```

**Review by calebcordry — APPROVED** [review]

**Comment by powerivq** [comment]
Adding @rcebulko for approval of build-system/test-configs/forbidden-terms.js

We need to allowlist amp-sticky-ad for extension feature because we are making it an alias of amp-ad.

**Comment by powerivq** [comment]
Adding @lannka for canary-config and prod-config.

**Review by lannka — APPROVED** [review]

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js [hunk 2]
```diff
@@ -112,6 +114,41 @@ class AmpStickyAd extends AMP.BaseElement {
...
+      return null;
+    }
+
+    ad.setAttribute('sticky', 'bottom');
+    this.element.parentElement.replaceChild(ad, this.element);
+    return Services.extensionsFor(this.win)
+      .loadElementClass('amp-ad', '0.1')
+      .then((AmpAd) => {
+        const impl = new AmpAd(ad);
+        impl.upgradeCallback();
```
Comment (jridgewell) [inline extensions/amp-sticky-ad/1.0/amp-sticky-ad.js:147]: Why are you calling `upgradeCallback` manually?

**Review by jridgewell — CHANGES_REQUESTED** [review]

File: extensions/amp-sticky-ad/1.0/amp-sticky-ad.js (hunk 2 above)
Comment (powerivq) [reply extensions/amp-sticky-ad/1.0/amp-sticky-ad.js:147]: I thought that an element cannot be upgraded the second time. Is that wrong?

Comment (powerivq) [reply extensions/amp-sticky-ad/1.0/amp-sticky-ad.js:147]: @jridgewell tested and it does upgrade again. Modified to not call upgrade manually.

**Review by jridgewell — APPROVED** [review]

**Pull request merged**